nodejs/eks/cluster.ts

```diff
@@ -1304,7 +1308,7 @@ export interface ClusterOptions {
 export interface FargateProfile {
     /**
      * Specify a custom role to use for executing pods in Fargate. Defaults to creating a new role
-     * with the `arn:aws:iam::aws:policy/AmazonEKSFargatePodExecutionRolePolicy` policy attached.
+     * with the `arn:${getPartition()}:iam::aws:policy/AmazonEKSFargatePodExecutionRolePolicy` policy attached.
```

Comment:
For the comment, let's use a less implementation-specific placeholder

```suggestion
     * with the `arn:[partition]:iam::aws:policy/AmazonEKSFargatePodExecutionRolePolicy` policy attached.
```